packages/react-core/src/next/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+import { css } from '@patternfly/react-styles';
+import { Menu, MenuContent, MenuProps } from '../../../components/Menu';
+import { Popper } from '../../../helpers/Popper/Popper';
+
+export interface DropdownProps extends MenuProps {
+  /** Anything which can be rendered in a dropdown. */
+  children?: React.ReactNode;
+  /** Classes applied to root element of dropdown. */
+  className?: string;
+  /** Renderer for a custom dropdown toggle. Forwards a ref to the toggle. */
+  toggle?: (toggleRef: React.RefObject<any>) => React.ReactNode;
```

Comment:
This is marked as optional, but when I try removing it from an example I get `TypeError: _ is not a function`.

Reply:
good catch. it is required.